databuilder/extractor/presto_view_metadata_extractor.py

```diff
@@ -0,0 +1,132 @@
+import base64
+import json
+import logging
+
+from pyhocon import ConfigFactory, ConfigTree  # noqa: F401
+from typing import Iterator, Union, Dict, Any  # noqa: F401
+
+from databuilder import Scoped
+from databuilder.extractor.base_extractor import Extractor
+from databuilder.extractor.sql_alchemy_extractor import SQLAlchemyExtractor
+from databuilder.models.table_metadata import TableMetadata, ColumnMetadata
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+class PrestoViewMetadataExtractor(Extractor):
+    """
+    Extracts Presto View and column metadata from underlying meta store database using SQLAlchemyExtractor
```

Comment:
It will be good to mention that the extractor doesn't require a separate table model but just reuse the existing TableMetadata.